com.civicrm.hrjobroles/js/src/job-roles/controllers/hr-job-roles-controller.js

```diff
@@ -238,7 +239,7 @@ define([
                     $scope.edit_data[role_id]['funders'] = [];
 
                     // Split data from the stored funder contact IDs
-                    var funder_contact_ids = data['funder'].split("|");
+                    var funder_contact_ids = $filter('isNotUndefined')(data['funder'].split("|"));
```

Comment:
Why introducing new $filters instead of creating simple filtering functions in the controller, using either "vanilla javascript" or lodash to filter the arrays ?